### PR TITLE
Add `futures_01` feature

### PR DESCRIPTION
**Pull request by Karrq**
Bump version number
Update `README.md`
Add support for futures 0.1 in `futures_01` module
Add test for support of futures 0.1

### CODE REVIEW

File: src/futures_01.rs [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                    $enum_name::$head_variant(ref mut x) => x.poll(),
+                    $(
+                        $enum_name::$tail_variants(ref mut x) => x.poll(),
+                    )*
+                }
+            
+            }
+        }
+
+        #[cfg(feature = "sink")]
```
Comment (glebpom) [inline src/futures_01.rs]: I think this is not required, since it's not an optional feature in 0.1

File: Cargo.toml [hunk 2]
```diff
@@ -1,20 +1,21 @@
...
 categories = ["asynchronous"]
 license = "MIT"
 homepage = "https://github.com/glebpom/one-of-futures"
-documentation = "https://docs.rs/one-of-futures/0.1.1/one-of-futures/"
+documentation = "https://docs.rs/one-of-futures/0.1.2/one-of-futures/"
 repository = "https://github.com/glebpom/one-of-futures"
 readme = "README.md"
 
 [dependencies]
 futures-core = "0.3.1"
```
Comment (glebpom) [inline Cargo.toml]: since we have futures 0.1 support now, future 0.3 should be also optional, enabled by default.

**Review by glebpom — CHANGES_REQUESTED** [review]
Thanks for your PR. Could you please also add travis build configuration with futures 0.1 support?

**Comment by Karrq** [comment]
> Could you please also add travis build configuration with futures 0.1 support?

I'm afraid I don't know how to do it. I'd need to look into it if you want me to do it.

**Comment by glebpom** [comment]
No problem, I'll do it separately

**Pull request merged**